src/System.Linq.Expressions/tests/New/TypeExtensions.cs

```diff
@@ -67,9 +67,14 @@ internal static bool IsByRefParameter(this ParameterInfo pi)
             string name,
             Type[] types)
         {
-            var method = type.GetAnyStaticMethod(name);
-
-            return method.MatchesArgumentTypes(types) ? method : null;
+            foreach (var method in type.GetRuntimeMethods())
```

Comment:
BTW: this is not a change to product code. For some reason there is copy of TypeExtensions in the test files. This change actually makes the copy consistent with the product code. A future PR should probably remove this file, but I hesitate to make too many changes in this file.